tck/bridge-tck-main-portlet/src/main/webapp/resources/test/resourcesRenderedInHeadTest.js

```diff
@@ -0,0 +1,81 @@
+(function() {
+
+	function getTestHeadElementId(element) {
+
+		var testHeadElementId = null;
+		var testHeadElementIds = window.getTestHeadElementIds();
+
+		for (var i = 0; i < testHeadElementIds.length; i++) {
+
+			if ((element.id && element.id.indexOf(testHeadElementIds[i]) > -1) || (element.src &&
+					element.src.indexOf('javax.faces.resource') > -1 &&
```

Comment:
@stiemannkj1 This JS assumes that the portlet container implementation does not have friendly and/or obfuscated URLs. Is there another way to check?